spec/controllers/api/v1/profiles_controller_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'rails_helper'
+
+module Api
+  module V1
+    describe ProfilesController do
+      render_views
+      let(:user) { create(:user) }
+
+      before :each do
+        request.env["HTTP_AUTHORIZATION"] = ActionController::HttpAuthentication::Token.encode_credentials(user.authentication_token)
+      end
+
+      describe "#show" do
+        it 'loads the current users profile' do
+          xhr :get, :show, id: 'me'
+          expect(assigns(:profile)).to eql(user)
+        end
+      end
+
+      describe "#update" do
+        it 'updates the users profile' do
+          new_attributes = {
+            name: 'new name',
+            email: 'new@example.com',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.